### DISCUSSION TITLE
Ajuda

### DISCUSSION THREAD

**Discussion by Math-Lira** [frontendbr/forum#2263] [Perguntas e Respostas] [answered]
Eu fui subir um projeto para o GitHub só que não estou conseguindo prosseguir com o seguinte erro:

$ git push -u origin main
error: src refspec main does not match any
error: failed to push some refs to 'https://github.com/Math-Lira/apis.git'

Como posso resolver ?

**Comment by igorlvicente** [comment, marked as answer]
https://stackoverflow.com/questions/4181861/message-src-refspec-master-does-not-match-any-when-pushing-commits-in-git

**Comment by felquis** [comment]
Sua dúvida é respondida por este comentário https://github.com/frontendbr/forum/discussions/1924#discussioncomment-932595

### Dica

Sempre procure pelo erro no fórum para ver se alguém já perguntou algo parecido.
